app/livechat/server/methods/forwardStatus.js

```diff
@@ -0,0 +1,13 @@
+import { Meteor } from 'meteor/meteor';
+import { hasPermission } from '../../../authorization';
+import { Livechat } from '../lib/Livechat';
+
+Meteor.methods({
+	'livechat:forwardstatus'(roomId, status) {
```

Comment:
As requested before, use `Camel Case` style.
In addition, it would be better renaming this method:

```suggestion
	'livechat:updateForwardStatus'(roomId, status) {
```